pennylane_lightning_gpu/src/bindings/Bindings.cpp

```diff
@@ -26,6 +26,8 @@
 #include "DevTag.hpp"
 #include "DevicePool.hpp"
 #include "Error.hpp"
+#include "MPIManager.hpp"
+#include "StateVectorCudaMPI.hpp"
```

Comment:
We also may want to conditionally allow compilation of these, with the mpi4py headers, only if required.

Reply:
done!